web-marketplace/src/pages/ChooseCreditClass/hooks/useGetCreditClassOptions.tsx

```diff
@@ -22,68 +27,44 @@ function useGetCreditClassOptions(): {
   creditClassOptions: CreditClassOption[];
   loading: boolean;
 } {
-  const [loading, setLoading] = useState(true);
-  const [creditClassOptions, setCreditClassOptions] = useState<
-    CreditClassOption[]
-  >([]);
+  const graphqlClient =
```

Comment:
It's probably best to view the file directly rather than look at the diff since it has been completely rewritten.